src/cpp/session/SessionConsoleProcessInfo.cpp

```diff
@@ -49,15 +50,15 @@ ConsoleProcessInfo::ConsoleProcessInfo(
          const std::string& handle,
          const int terminalSequence,
          TerminalShell::TerminalShellType shellType,
-         ChannelMode channelMode,
-         const std::string& channelId,
-         int maxOutputLines)
+         bool altBufferActive,
+         core::FilePath cwd,
```

Comment:
Nit: We usually pass a FilePath as a const reference to avoid the overhead of creating a copy (obviously negligible in this case)